conf.py

```diff
@@ -157,11 +161,39 @@
     'custom.css',
     'header.css',
     'github_issue_links.css',
-    'furo_colors.css'
+    'furo_colors.css',
+    'footer.css'
 ]
 html_css_files.extend(custom_html_css_files)
 
-html_js_files = ['header-nav.js']
+html_js_files = ['header-nav.js', 'footer.js']
 if 'github_issues' in html_context and html_context['github_issues'] and not disable_feedback_button:
     html_js_files.append('github_issue_links.js')
 html_js_files.extend(custom_html_js_files)
+
+#############################################################
+# Display the contributor
+
+def get_contributors_for_file(github_url, github_folder, pagename, page_source_suffix):
+    parsed_url = urlparse(github_url)
+    path_parts = parsed_url.path.split('/')
+    username = path_parts[1]
+    repository = path_parts[2]
+    filename = f"{pagename}{page_source_suffix}"
+    repo = Repo(".") 
+    commits = repo.iter_commits('--all', paths=filename)
```

Comment:
We should make it configurable if you want to display all contributors, or only contributors within a certain time frame.
You can use the `since=` parameter for that.

Since this should be customisable by the user, we need a variable in `custom_conf.py` for it, for example, `display_contributors_since`.
If this is not set (or set to an empty string), we don't need to do anything different.
But if it is set, we should call this instead:

```suggestion
    commits = repo.iter_commits('--all', paths=filename, since=display_contributors_since)
```